templates/scenarios/js/sso-tab/aad.manifest.template.json.tpl

```diff
@@ -1,7 +1,10 @@
+{{=<% %>=}}
 {
     "id": "${{AAD_APP_OBJECT_ID}}",
     "appId": "${{AAD_APP_CLIENT_ID}}",
-    "name": "sso-tab-aad",
+<%={{ }}=%>
+    "name": "{{appName}}-aad",
```

Comment:
What about using `<%appName%>`? You don't need to surround the placeholder with `<%={{ }}=%>` and `{{=<% %>=}}`.

Reply:
ok, as discussed with @hund030 , he will update the generator to use other pattern as the placeholder.